app/views/devise/sessions/new.html.erb

```diff
@@ -34,7 +35,12 @@
         } %>
       </div>
     <% end -%>
-    <%= f.submit t('account.sign_in'), class: 'buttonGhostM marginTop' %>
+
+    <% if @display_recaptcha %>
+      <%= recaptcha_tags id: 'recaptcha-form', callback: 'recaptcha_verified', expired_callback: 'recaptcha_expired' %>
```

Comment:
`recaptcha_verified` and `recaptcha_expired` are the names of the javascript functions that run when a recaptcha is completed/expires. Recaptchas expire after about 2 minutes.